pkgs/development/libraries/qscintilla/default.nix

```diff
@@ -14,9 +14,15 @@ stdenv.mkDerivation rec {
     sha256 = "04678skipydx68zf52vznsfmll2v9aahr66g50lcqbr6xsmgr1yi";
   };
 
-  buildInputs = if withQt5 then [ qtbase ] else [ qt4 ];
+  buildInputs = if withQt5 then ([ qtbase ] ++ (stdenv.lib.optional stdenv.isDarwin qtmacextras)) else [ qt4 ];
   nativeBuildInputs = [ unzip ] ++ (if withQt5 then [ qmake ] else [ qmake4Hook ]);
 
+  # Fix Xcode 8 compilation problem
+  xcodePatch = fetchurl { url = "https://raw.githubusercontent.com/Homebrew/formula-patches/a651d71/qscintilla2/xcode-8.patch";
```

Comment:
either add this directly in the `patches` list, or put it in a `let in` expression.

Reply:
Good idea.